bundle.yaml

```diff
@@ -1,21 +1,85 @@
+# The machine constraints for each service in this bundle
+# have been commented out so you don't run into quota
+# problems on public clouds. Modify and uncomment the
+# constraints: lines for each service to reflect your
+# deployment before moving to production.
+#
 series: xenial
-services:
-  kubernetes:
-    charm: cs:xenial/kubernetes-8
-    options:
-      version: v1.3.4
+machines:
+  0:
+  1:
+  2:
+  3:
+applications:
+  "kubernetes-master":
+    charm: "cs:~containers/kubernetes-master-3"
+    num_units: 1
+    to:
+      - 1
+    annotations:
+      "gui-x": "800"
+      "gui-y": "850"
+  flannel:
+    charm: "cs:~containers/flannel-4"
+    annotations:
+      "gui-x": "450"
+      "gui-y": "750"
+  "kubeapi-load-balancer":
+    charm: "cs:~containers/kubeapi-load-balancer-2"
+    num_units: 1
+    to:
+      - 0
+    expose: true
+    annotations:
+      "gui-x": "450"
+      "gui-y": "250"
+  easyrsa:
+    charm: "cs:~containers/easyrsa-2"
+    num_units: 1
+    to:
+      - lxd:0
+    annotations:
+      "gui-x": "450"
+      "gui-y": "550"
+  "kubernetes-worker":
+    charm: "cs:~containers/kubernetes-worker-3"
```

Comment:
We also published a new kubernetes-worker with fixes. Can you change this to 4 as well?